video/filter/vf_vapoursynth.c

```diff
@@ -699,6 +700,8 @@ static int reinit_vs(struct priv *p, struct mp_image *input)
     p->vsapi->propSetFloat(vars, "container_fps", container_fps, 0);
     p->vsapi->propSetFloat(vars, "display_fps", display_fps, 0);
     p->vsapi->propSetIntArray(vars, "display_res", display_res, 2);
+    p->vsapi->propSetData(vars, "user_data", p->opts->user_data,
```

Comment:
Have you considered runtime changes of this option?

Reply:
Fixed

Reply:
Yes, something like that. The question is to you if it is useful. In my opinion ability to update parameters during runtime may be useful, but if you think it is not needed we can go with static data.

Reply:
That could be useful indeed. However, I think that's a separate, and bigger change (e.g. we need a "template" syntax to represent the dynamic data. Also potentially need escaping invalid characters, etc) that warrant a PR on its own. I think we can wait until someone requests and then decide, which by then we'd have better understanding of the scope.